app/controllers/api/base.rb

```diff
@@ -8,8 +8,6 @@ def parameters(request, _)
       end
     end
   end
-  # Need to add file parameters to this so it doesn't try (and fail) to make images json
-  FILTERED_PARAMS = Rails.application.config.filter_parameters + [:file]
```

Comment:
**We need this**.

Without it, we 500 when uploading a file because it tries to log the file as JSON - and errors, since the file is binary.

Potential solution: inline this, replacing the line in this file that references `FILTERED_PARAMS`

```ruby
GrapeLogging::Loggers::FilterParameters.new(GrapeLogging::FILTERED_PARAMS)
```

_p.s. sorry there aren't tests for this, I don't know how to test it_

Reply:
Looks like it was only a no-op for me because of Spring (the app being initialized already). I'll just remove the change for now.